optuna/storages/rdb/storage.py

```diff
@@ -848,6 +850,20 @@ def _merge_trials_orm(
 
         return result
 
+    @staticmethod
+    def _set_default_engine_kwargs_for_mysql(url, engine_kwargs):
+        # type: (str, Dict[str, Any]) -> None
+
+        # Skip if RDB is not MySQL.
+        if not url.startswith('mysql'):
+            return
+
+        # Do not overwrite value.
+        if 'pool_pre_ping' in engine_kwargs:
+            return
+
+        engine_kwargs['pool_pre_ping'] = True
```

Comment:
I think that it could be helpful for future readers to leave a brief comment about why setting this option is needed when using MySQL backend.

Reply:
Thank you for your suggestion. That makes sense.
I added the comment as follows:

```python
        # If True, the connection pool checks liveness of connections at every checkout.
        # Without this option, trials that take longer than `wait_timeout` may cause connection
        # errors. For further details, please refer to the following document:
        # https://docs.sqlalchemy.org/en/13/core/pooling.html#pool-disconnects-pessimistic
```

Reply:
Thanks!